pkg/api/graph/graph.go

```diff
@@ -122,7 +122,7 @@ var DefaultNamer Namer = namer{}
 func (namer) ResourceName(obj interface{}) string {
 	switch t := obj.(type) {
 	case uniqueNamer:
-		return t.UniqueName()
+		return t.UniqueName().UniqueName()
```

Comment:
Sure ... items like BuildConfigNode, ImageStreamTagNode, etc. were not passing as instances of the `uniqueNamer` interface, where the `UniqueName()` method has a return of type `UniqueName`.

So I had to pick between getting consistent with a return type of `UniqueName`.  Assuming that the specialized type was created for a good reason, I also based picking `UniqueName` over string on the fact that similar k8s resources (see https://github.com/openshift/origin/blob/master/pkg/api/kubegraph/nodes/types.go) used a return type of `UniqueName`.

Bottom line - had to make things consistent.  We could go back and change the return type to `string` and make the requisite changes in that direction.